src/components/Header/Logo.js

```diff
@@ -1,22 +1,29 @@
 import { Box } from 'grommet'
+import { Anchor } from 'components/shared/Anchor'
+import { SrOnly } from 'components/shared/SrOnly'
 import styled, { css } from 'styled-components'
 import { LogoSvg } from '../../images/logo.svg'
 
 // NOTE: Set the prop 'light' to true for the dark background
 
 const Wrapper = styled(Box)`
```

Comment:
I don't think we need to use a wrapper here. `LogoAnchor` makes more sense to me.

Reply:
Renamed, thank you!